virt/virt-4-10-release-notes.adoc

```diff
@@ -122,6 +122,12 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-10-known-issues"]
 == Known issues
 
+//Known issues at time of 4.10 release
+//BZ-2054650
+* The web console does not display templates that are deployed to a custom namespace. Only templates deployed to the default namespace will display in the web console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2054650[*BZ#2056623*])
```

Comment:
Suggestion: s/templates/virtual machine templates for the first instance of templates.

Reply:
Updated the content.